rwatch/graphics/gbitmap.c

```diff
@@ -265,10 +266,10 @@ GBitmap *gbitmap_create_with_resource(uint32_t resource_id)
     return gbitmap_create_from_png_data(png_data, png_data_size);
 }
 
-GBitmap *gbitmap_create_with_resource_app(uint32_t resource_id, uint16_t slot_id)
+GBitmap *gbitmap_create_with_resource_app(uint32_t resource_id, const File* file)
```

Comment:
Currently there's `const File` which has current-offset information (kind of like a file descriptor), but we also use it to refer to the abstract idea of a file (i.e., starting from 0).  Do we want a thing that comes in two parts -- `fs_find_file` turns a name into a `struct file`, and then `fs_open` turns a `struct file` into a `struct fd` (and then on a `struct fd`, you could `fs_read` and `fs_seek`)?